tests/test_concurrency.py

```diff
@@ -0,0 +1,29 @@
+import sys
+
+import pytest
+
+from httpx import AsyncioBackend, HTTPVersionConfig, SSLConfig, TimeoutConfig
+
+
+@pytest.mark.skipif(
+    sys.version_info < (3, 7),
+    reason="Requires Python 3.7+ for AbstractEventLoop.start_tls()",
+)
```

Comment:
Perhaps use `pytest.mark.xfail` instead to make sure the test *does* fail on 3.6?